Representable/representable
# add resources page

**changes**
- added a resource page according to design, with link in Learn More dropdown
**to test**
- python manage.py runserver
- go to http://127.0.0.1:8000/resources/
- check for errors in formatting / links
**screenshots**
desktop
![image](https://user-images.githubusercontent.com/41943646/107700398-87236680-6c5b-11eb-9a22-db7d8a3be259.png)
ipad
![image](https://user-images.githubusercontent.com/41943646/107700451-973b4600-6c5b-11eb-94e4-1859ef3b0108.png)
iphone X
![image](https://user-images.githubusercontent.com/41943646/107700498-a6ba8f00-6c5b-11eb-9991-e0aeeba1d013.png)
